content/en/docs/contribute/style/hugo-shortcodes/index.md

```diff
@@ -37,7 +37,7 @@ The valid values for `state` are:
 
 ### Feature state code
 
-Below is the template code for each available feature state.
+Given below is the template code for each feature state.
```

Comment:
I think we can remove this sentence altogether. It adds no value.

Reply:
Agreed. I will remove it.